content/common/cli-reference.md

```diff
@@ -1208,3 +1209,28 @@ TRIGGER OPTIONS:
 {{ blockEnd }}
 
 {{ blockEnd }}
+
+### CLI Stability Table
+
+CLI commands have three phases that indicate levels of stability:
+
+- `experimental`: These commands are experiments and may or may not be available in later versions of the CLI.
+- `stabilizing`: These commands have moved out of the `experimental` phase and we are now in the active process of stabilizing them. This includes updating flags, command output, errors, and more.
+- `stable`: These commands have moved out of the `stablizing` phase and will not change until the next major version release.
+
+| Command                         | Stability    |
```

Comment:
Is it worth saying which version of Spin the table refers to?

Reply:
Added the spin version tab like the other sections in this doc.